### ISSUE TITLE
[Package Request] Navidrome

### ISSUE THREAD

**Issue by rfporter** [SynoCommunity/spksrc#4973]
* Name: Navidrome
* Description: Open source music server
* Website: https://www.navidrome.org/
* Software documentation: https://www.navidrome.org/docs/
* Build/Installation documentation: https://www.navidrome.org/docs/installation/
* Source code:  https://github.com/navidrome/navidrome/
* License: GPL-3.0 License

**Comment by hgy59** [comment]
@rfporter I tried to build a navidrome package.
The frontend builds successfully when native/nodejs is updated to v16.
Unfortunately the navidrome executable fails to build (even with go downgraded to v1.16).
Error is: `scanner/metadata/taglib/taglib.go:22:15: undefined: Read`

I also tried to build with `make single`. Therefore I had to install docker in docker and to patch the Makefile to run docker with sudo and to mount the host folder to /workdir for goreleaser, but all cross compilation failed with `unknown option -m64`.

We could try to use the prebuilt binaries, but this might fail for incompatible clib.
